### ISSUE TITLE
Undefined stream error on login

### ISSUE THREAD

**Issue by bodom0015** [whole-tale/ngx-dashboard#6]
**I'm submitting a ...**  (check one with "x")
```
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  <!-- Please check the repository for a similar issue or PR before submitting -->
[ ] Support request => <!-- Please check the repository for a similar issue or PR before submitting -->
[ ] Feature request
[ ] Documentation issue or request
```

**Current behavior**
Intermittently receive a seemingly innocuous error upon logging in: `You provided 'undefined' where a stream was expected.`

**Expected/desired behavior**
No error shown on successful login

**Minimal reproduction of the problem with instructions**
1. Login to the WholeTale Dashboard

**What is the motivation / use case for changing the behavior?**
Current behavior is distracting / annoying

**Environment**
* **Angular version:** X.Y.Z

* **Browser:**
- [x] Chrome (desktop) version XX
- [ ] Chrome (Android) version XX
- [ ] Chrome (iOS) version XX
- [ ] Firefox version XX
- [ ] Safari (desktop) version XX
- [ ] Safari (iOS) version XX
- [ ] IE version XX
- [ ] Edge version XX

* **For Tooling issues:**
- Node version: XX  
- Platform:  Mac

* Others:

**Comment by ThomasThelen** [comment]
I haven't been able to reliable reproduce this; could have been fixed by #86

**Comment by bodom0015** [comment]
Haven't seen this in awhile. Closing this for now
